scripts/globals/spells/trust/monberaux.lua

```diff
@@ -32,13 +45,9 @@ spellObject.onMobSpawn = function(mob)
         mobArg:setTP(0)
     end)
 
-    -- TODO: Monberaux's moves all come out with ACTION_ITEM_START and ACTION_ITEM_FINISH in their start/end action
-    --     : packets, so we either need to fully implement a mob/trust's ability to use items (faking inventory space
-    --     : etc.), or we need more workarounds like below to intercept and change the messages.
-    --     : Currently, we only have enough bindings and triggers to change the action end packet.
-    -- mob:addListener("WEAPONSKILL_USE", "MONBERAUX_ITEM_WEAPONSKILL_USE", function(mobArg, target, wsid, tp, action)
-    --     action:setCategory(5) -- ACTION_ITEM_FINISH
-    -- end)
+    mob:addListener("WEAPONSKILL_USE", "MONBERAUX_WS", function(mobArg, targetArg, skillid, spentTP, action)
```

Comment:
This probably wants the above comment kept/adjusted to explain _why_ we need this